docs/authoring.md

```diff
@@ -183,6 +183,7 @@ The `.tp-config.json` file contains a JSON object that may have the following in
 |`supportedProtocols`| [String] | A list with all the protocols this template supports.
 |`parameters`| Object[String, Object] | An object with all the parameters that can be passed when generating the template. When using the command line, it's done by indicating `--param name=value` or `-p name=value`.
 |`parameters[param].description`| String | A user-friendly description about the parameter.
+|`parameters[param].default`| String | Default value of this parameter if not set in the start command.
```

Comment:
I'm not sure about type `String`. Actually `String`, `Integer`, `Boolean` will work. But from CLI all parameter values are `String`, so i set here only `String`